forc/Cargo.toml

```diff
@@ -25,6 +25,7 @@ forc-pkg = { version = "0.15.1", path = "../forc-pkg" }
 forc-util = { version = "0.15.1", path = "../forc-util" }
 fs_extra = "1.2"
 fuel-asm = "0.5"
+fuel-crypto = "0.5"
```

Comment:
It looks like `fuel-tx` already has `fuel-crypto` as a dependency, so there should be no need to add `fuel-crypto` as an explicit dependency here!

Reply:
I am not entirely sure how to reference a dependency of a dependency without modifying the `fuel-tx` crate to expose `fuel-crypto`. It seems that `fuel-tx` also has `fuel-asm` as a dependency, yet it is also listed as a separate dependency here.

How should I go about this without adding `fuel-crypto` explicitly here?

Reply:
Hmmm good point. This may not be possible after all? @sezna, @adlerjohn thoughts?

Reply:
Huh, you actually appear to be correct @seem-less, `fuel-crypto` is not re-exported by `fuel-tx`! Nevermind then, you can leave `fuel-crypto` as a dependency.

As for why `fuel-asm` is a dependency...that's code debt that was never fixed 😂. Feel free to open another PR to clean it up.